ppr-ui/src/components/dialogs/SecuredPartyDialog.vue

```diff
@@ -311,6 +311,17 @@ export default defineComponent({
   border: 1px solid white;
 }
 
+.companyRowNoHover {
```

Comment:
snake case as the other classes. 
for `companyrow` as well 👍

Reply:
Kinda confused on when to use which case. The majority of classes in this file are camel case. hmmmmm

Reply:
It's a valid question. The formatting is a matter of preference and consistency.
We didn't build this app originally and there is many instances of a few different ways to do things. 
This is one of those instances. 

This type of thing, we sort of try and fix as we go, so it's not on you to catch these things but the sort of thing we can spot and help with during code review.

Reply:
If you feel the fix is large and quite out of scope, than by all means you can leave it but just commented so that you are aware of snake case being the defacto preference for element identifiers (id's, classes, test-ids' etc)

Reply:
Gotcha, thanks for clarifying! I'll make that change